dvc/dependency/param.py

```diff
@@ -2,12 +2,11 @@
 from collections import defaultdict
 
 import dpath.util
-import toml
```

Comment:
@pmrowla, [`tomlkit`](https://github.com/sdispater/tomlkit) preserves ordering and comments.